CMakeLists.txt

```
@@ -39,6 +39,11 @@ check_cxx_compiler_flag(-fopenmp-simd CXX_HAVE_OMP_SIMD)
 # Build options
 option(BUILD_DALI_NODEPS "Disable components that require extra external libraries to be present in the system. Effectively, it builds only the DALI core and kernel libraries")
 option(LINK_DRIVER "Links directly with libcuda.so instead of dlopen it at runtime" OFF)
+option(WITH_DYNAMIC_CUDA_LIBS "Links CUDA libraires dynamically (NPP, nvJPEG, cuFFT)" OFF)
+
+if (WITH_DYNAMIC_CUDA_LIBS)
+  add_definitions(-DWITH_DYNAMIC_CUDA_LIBS=1)
+endif(WITH_DYNAMIC_CUDA_LIBS)
```

```suggestion
endif()
```
I believe it's not necessary anymore for a few years ;) Same thing applies to the rest of `ifs` below.

---

Done